### PR TITLE
Added the three ITU Regions for radio frequency scanning.  

### PR DESCRIPTION
**Pull request by DwaineGarden**
This will allow people to scan FM and AM with the right frequency range for their area.

### CODE REVIEW

**Comment by jamesgol** [comment]
This is cool, but instead of ITU Region 1, 2, 3 I think it would be better to have a more obvious description of the areas.  I had to google them to see what was what and I'm sure the common user won't know what region they are in either.

It also could be possible to autolocate the user, but it's non trivial to do.  If there was network access it's much easier, but that isn't something we can depend on.

File: Radio/jni/com_bonovo_radio.cpp [hunk 1]
```diff
@@ -22,8 +22,8 @@
 #define HANDLE_CTRL_DEV_MAJOR		230
 #define HANDLE_CTRL_DEV_MINOR		0
 
-#define IOCTL_HANDLE_GET_BRIGHTNESS		    _IO(HANDLE_CTRL_DEV_MAJOR, 1)
-#define IOCTL_HANDLE_GET_WINCEVOLUME		_IO(HANDLE_CTRL_DEV_MAJOR, 2)
+#define IOCTL_HANDLE_GET_BRIGHTNESS	    _IO(HANDLE_CTRL_DEV_MAJOR, 1)
+#define IOCTL_HANDLE_GET_WINCEVOLUME	    _IO(HANDLE_CTRL_DEV_MAJOR, 2)
```
Comment (jamesgol) [inline Radio/jni/com_bonovo_radio.cpp:26]: Please try to limit whitespace changes

**Comment by DwaineGarden** [comment]
Use of GPS to determine the location of the radio.   That would be cool.

I can add countries to the descriptions.

File: Radio/jni/com_bonovo_radio.cpp (hunk 1 above)
Comment (DwaineGarden) [inline Radio/jni/com_bonovo_radio.cpp:26]: Sorry....

**Comment by jamesgol** [comment]
What ever happened to this?  I never noticed it was closed so I always assumed it got merged in

**Comment by DwaineGarden** [comment]
I should try and use Google location services to pin point the radio location automatically. Then select the itu region.

**Comment by jamesgol** [comment]
Sounds like a great long term goal, but I think just having the manual choices is good enough for now.  If there's no major problems with the code I'll see about committing it without the whitespace changes.

What we are working with isn't perfect so any improvements we can make are worthwhile.

**Comment by lkernan** [comment]
I think if it works ok then lets pull it in. It can be improved from there.

**Comment by DwaineGarden** [comment]
Ok... Pull it.   I can work on getting the location services working to auto select the ITU Region.

I also have been working on getting the wx radio frequencies added to the radio application.   The silabs tuner is great....

**Comment by jamesgol** [comment]
Did you miss commiting some parts of this?  I had to fix up a bunch of stuff to get it to work and even compile.  

Need to do a bit more testing and I'll commit

**Comment by jamesgol** [comment]
You should really update your repo and/or commit those changes in a separate branch.  There are going to be enough differences that it's going to be a pain to merge

**Comment by DwaineGarden** [comment]
Oh come on GITHUB.   These changes should of not been a part of this pull.  

Add the ability for WX Radio …  

 65ffa4e  
  @DwaineGarden   Update com_bonovo_radio.cpp …  

 ce00080  
  @DwaineGarden   Update com_bonovo_radio.cpp …  

 d380e2f  
  @DwaineGarden   Update com_bonovo_radio.cpp …

**Comment by DwaineGarden** [comment]
GitHub is shit.

**Comment by jamesgol** [comment]
Lol

**Comment by jamesgol** [comment]
You need to create a separate branch for each thing you are working on, PR are based on the branch.  In this case it is the master repo you forked.

**Comment by DwaineGarden** [comment]
Can you just pull the changes from June.  Not todays.  I was working on the WX radio functions.  Having:
FM
AM
WX

**Comment by jamesgol** [comment]
I cant/wont just merge it as is.  The code wasn't complete and non-functional.  I've got it mostly working but there are still some issues I'm tracking down.

**Comment by DwaineGarden** [comment]
Ok..... Why can't it just understand a pull request is a pull request for a branch.  So I could have like 5 pull requests for a specific branch.

**Pull request closed without merging**